### ISSUE TITLE
Cloud campaign

### ISSUE THREAD

**Issue by kristinashu** [mozilla/foundation.mozilla.org#1524]
**Static page:**
~An open letter asking _retailers_ to stop selling the toy (copy in email). We need one image for website page and resized for social share (Twitter/Facebook).~  This will now be a blog post

**Petition page:**
A separate general petition for _consumers_ to sign. We need one image for website page and resized for social share (Twitter/Facebook). [need copy]

No specific visuals in mind so up to Natalie! But images for the two should look similar but with slightly different content.

Launches next week [Sara to confirm exact date asap].

Driver: Sara (will share with Sara to share with others)
Accountable: Ashley 
Responsible: Natalie for graphic, Kevin for copy)
Consulted: Marshall, Abigail, Chris R, Erica D, 
Informed: Policy, Advocacy, Marketing and Comms

cc @xmatthewx

**Comment by kristinashu** [comment]
I've updated the first comment. The main update is that this is for a campaign asking retailers to stop selling (no petition, just an open letter). Later, we will have a more general petition that people can sign but not right now.

**Comment by kristinashu** [comment]
cc @alanmoo to create the petition form

**Comment by gideonthomas** [comment]
@kristinashu what is the title of the page going to be?

**Comment by kristinashu** [comment]
Pinged you in slack!

**Comment by gideonthomas** [comment]
Petition form and page created
TODO - copy to be added to the page in the CMS and published

Share buttons created
TODO - update the copy and images for the share buttons in ShareProgress

Let me know if you need anything else.

**Comment by natalieworth** [comment]
@gideonthomas @kristinashu The Cloud Pets illustrations have been updated and added in redpen.
https://redpen.io/p/hkd825b18079a13112

I've also added exported files here https://drive.google.com/drive/u/1/folders/16eTEjwqhu2cal2HOb0kkgXwLhMZAHMfn

But post in redpen if there are any issues with either. Sara was good with the 1st one (seems like that was everyone's favorite—and the least creepy for the subject matter).
